docs/guide/drag_and_drop.md

```diff
@@ -0,0 +1,36 @@
+# Drag and drop interface class
+
+In eZ Platform you are able to create a generic interface for drag and drop interactions that can be reused in many places.
+
+First prepare the HTML code structure that will be placed in a Twig template in the following way:
+
+```html
+<div class="items-container" data-placeholder="HTML template for placeholder"></div>
+```
+
+Next, add options in the same Twig template or in a JavaScript code that comes with the template following the convention:
+
+```javascript
+const draggable = new global.eZ.core.Draggable({
+        itemsContainer: HTMLElement of .items-container,
+        selectorItem: String,
+        selectorPlaceholder: String,
+});
+```
+
+For more information on creating Twig templates see [Templating basics](../guide/templates.md).
+
+## Options
+
+Full list of options:
+
+|Option|Description|Required|
+|------|-----------|--------|
+|`itemsContainer`|a reference to DOM node containing a draggable items|required|
```

Comment:
`a draggable item` (no "s")
Also, I think you can drop the "a" at the start of each description